db/schema.rb

```diff
@@ -13,17 +13,20 @@
 
 ActiveRecord::Schema.define(version: 20160815140302) do
 
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
   create_table "ahoy_events", force: :cascade do |t|
-    t.uuid     "visit_id",   limit: 16
+    t.integer  "visit_id"
```

Comment:
I am still trying to figure out the best solution to that, here are some concerns I have:
1 - the type of visit_id column depends on the Ahoy store being used, so I am not sure if it is a good idea to just override the column type in the migration
2 - with the migration scenario another problem may arise in case if we are updating already existing records - how can we convert UUID to and Integer types without ruining the relations for such records?
3 - I am not exactly sure how the database ends in an invalid state with the different column types for the visit_id vs visits.id

If you know how the Ahoy works in detail, please advise.
Of course, I can just add the migration to change the column type but I don't want to risk breaking anything :)